### PR TITLE
Solves #15

### PR DESCRIPTION
**Pull request by Morgenstern2573**
## What kind of change does this PR introduce?

Bug fix, feature, docs update, ...

Updates the project README

## What is the current behavior?

Please link any relevant issues here.
[#15](https://github.com/supabase-community/storage-go/issues/15)

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,29 +17,57 @@ package main
...
+	client := storage_go.NewClient("https://<project-reference-id>.supabase.co/storage/v1", "<project-secret-api-key>", nil)
 
-	// Get buckets
-	fmt.Println(client.ListBuckets())
+	// Create a new bucket
+	bucket, berr := client.CreateBucket("bucket-id", storage_go.BucketOptions{Public: true})
 
-	// Upload a file
+	if berr.Error != "" {
+    log.Fatal("error creating bucket, ", berr)
```
Comment (jackgris) [inline README.md]: The spaces here are correct to a go file, but sometimes a markdown file doesn't look well. Maybe changing some spaces will be fine.

**Review by jackgris — COMMENTED** [review]
The only detail is what I commented before, otherwise LGTM

File: README.md [hunk 2]
```diff
@@ -14,29 +17,57 @@ package main
...
 	file, err := os.Open("dummy.txt")
 	if err != nil {
 		panic(err)
 	}
 
 	resp := client.UploadFile("bucket-name", "file.txt", file)
 	fmt.Println(resp)
+
+	// Update Bucket
+	response, berr := client.UpdateBucket(bucket.Id, storage_go.BucketOptions{Public: true})
```
Comment (jerichosiahaya) [inline README.md]: I appreciate your work on this readme. I noticed that you're using `berr` as the variable name for errors in this readme. In the Go community, it's a convention to use `err` as a short, descriptive name for error variables. For the sake of consistency.

File: README.md (hunk 1 above)
Comment (Morgenstern2573) [reply README.md]: I'll sort that out, thanks for the feedback!

File: README.md (hunk 2 above)
Comment (Morgenstern2573) [reply README.md]: I'll make the changes soon, thanks for the review!

**Review by jackgris — APPROVED** [review]
LGTM

**Review by jerichosiahaya — APPROVED** [review]

**Pull request merged**